src/config/index.ts

```diff
@@ -168,6 +170,16 @@ export const REVIEW_OWNER_INDEX_PARAM = "?index=";
 export const UPDATE_REVIEW_OWNERS_PARAMS = REVIEW_OWNER_INDEX_PARAM + ID;
 export const ROUTE_PARAM_TRANSACTION_ID = "transactionId";
 export const ROUTE_PARAM_SUBMISSION_ID = "submissionId";
+export const LANDING_PAGE_QUERY_PARAM = "start";
+export const JOURNEY_QUERY_PARAM = "journey";
+
+export enum JourneyType {
+  register = "register",
+  update = "update",
+  remove = "remove",
+}
+
+export const JOURNEY_REMOVE_QUERY_PARAM = `?${JOURNEY_QUERY_PARAM}=${JourneyType.remove}`;
```

Comment:
Since we use this string in a couple of places at present and will use it in more in the future, thought we could have a common constant for it.